src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -102,8 +102,9 @@ private URI checkDockerHostScheme(URI dockerHost) {
     private String checkDockerCertPath(boolean dockerTlsVerify, String dockerCertPath) {
         if (dockerTlsVerify) {
             if (StringUtils.isEmpty(dockerCertPath)) {
-                throw new DockerClientException(
-                        "Enabled TLS verification (DOCKER_TLS_VERIFY=1) but certifate path (DOCKER_CERT_PATH) is not defined.");
+                return dockerCertPath;
+//                throw new DockerClientException(
```

Comment:
For using a custom SSLContext you would have to do two thing: 

1. disable usage of dockerCertPath in the config